services/knative/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+displayName: Knative
+description: Kubernetes-based platform to build, deploy, and manage modern serverless workloads
+category:
+  - general
+type: catalog
+scope:
+  - workspace
+overview: |-
+  TODO
+icon: TODO
```

Comment:
do we need to update that as well?

Reply:
looks like we should

Reply:
Who can help add some metadata here?

Reply:
I'm going to merge this as is to get it out for 2.1.1 RC, but we need to get this filled out before 2.1.1 GA next week. @alembiewski would you be able to fill this out, or ask someone who is familiar with knative?

Reply:
Sure, I will do it